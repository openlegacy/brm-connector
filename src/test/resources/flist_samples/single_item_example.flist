0 PIN_FLD_POID                      POID [0] 0.0.0.1 /account -1 0
0 PIN_FLD_INT_VAL                    INT [0] 1
0 PIN_FLD_PROGRAM_NAME               STR [0] "program"
0 PIN_FLD_PAYINFO                  ARRAY [0] allocated 2, used 2
1     PIN_FLD_POID                  POID [0] 0.0.0.1 /account -1 0
1     PIN_FLD_INHERITED_INFO   SUBSTRUCT [0] allocated 1, used 1
2         PIN_FLD_CC_INFO          ARRAY [0] allocated 6, used 6
3             PIN_FLD_PROVIDER_IPADDR    BINSTR [0] 5 C885939396
3             PIN_FLD_BUFFER         BUF [0] flag/size/offset 0x00 5 0 data: 
x000000  0102030405                         .....
3             PIN_FLD_RESIDENCE_FLAG    ENUM [0] 1
3             PIN_FLD_AMOUNT     DECIMAL [0] 1.1
3             PIN_FLD_DEBIT_EXP      STR [0] "XXXX"
3             PIN_FLD_DUE_DATE_T  TSTAMP [0] (1591142400) 03/06/2020 03:00:00:000 AM
