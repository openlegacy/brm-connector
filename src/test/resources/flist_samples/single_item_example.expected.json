{
  "pinFldPoid" : {
    "databaseNumber" : 1,
    "objectType" : "/account",
    "objectId" : -1,
    "objectRevision" : 0
  },
  "pinFldIntVal" : 1,
  "pinFldProgramName" : "program",
  "pinFldPayinfo" : [ {
    "pinFldPoid" : {
      "databaseNumber" : 1,
      "objectType" : "/account",
      "objectId" : -1,
      "objectRevision" : 0
    },
    "pinFldInheritedInfo" : {
      "pinFldCcInfo" : [ {
        "pinFldDebitExp" : "XXXX",
        "pinFldResidenceFlag" : 1,
        "pinFldAmount" : "1.1",
        "pinFldDueDateT" : "2020-06-03",
        "pinFldProviderIpaddr" : "yIWTk5Y=",
        "pinFldBuffer" : "AQIDBAU="
      } ]
    }
  } ]
}